src/run/interact.rs

```diff
@@ -115,7 +114,6 @@ impl<'a, M: Mode> Net<'a, M> {
   ///       \ /       \ /
   ///        |         |
   ///     b1 |         | b2
-  ///  
```

Comment:
was this line auto-removed? I'd like it kept for equal spacing

Reply:
Oh, I think I remember now – I believe rustfmt automatically removes this line unless it has trailing spaces. If you could just remove the blank line at the top, then.